pom.xml

```diff
@@ -274,6 +274,12 @@
 				<version>${assertj.version}</version>
 				<scope>test</scope>
 			</dependency>
+			<dependency>
+				<groupId>org.osgi</groupId>
+				<artifactId>org.osgi.test.assertj</artifactId>
```

Comment:
I don't think this is a good idea to put in the parent pom. It should only be in the project (like junit5) pom which can use the assertj project to test.

Reply:
But I'ved added it to dependencyManagement, not to the raw dependencies. Adding it to dependencyManagement simply sets the default scope and version for child projects *if* any of them decide to add the artifact to their own dependencies. It doesn't add it to the dependencies of all projects by default.

Reply:
My bad. The "local" view in the PR did not make it clear. I should have looked the the complete file.